Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/trees/Tree.tsx

```diff
@@ -14,11 +14,12 @@ import styles from './Tree.module.scss';
 
 interface ITreeProps {
     items: ITreeItemType[];
-    onSelect: (node: ITreeItemType) => void;
+    onSelect: (node: any) => void;
```

Comment:
No. Read how to make this type as 'ITreeItemType` or _un inheritant of ITreeItemType_